pkg/payload/payload.go

```diff
@@ -134,13 +133,15 @@ type metadata struct {
 }
 
 func LoadUpdate(dir, releaseImage, excludeIdentifier string, includeTechPreview bool, profile string,
-	capabilities capability.ClusterCapabilities) (*Update, error) {
+	knownCapabilities []configv1.ClusterVersionCapability) (*Update, error) {
 
 	payload, tasks, err := loadUpdatePayloadMetadata(dir, releaseImage, profile)
 	if err != nil {
 		return nil, err
 	}
 
+	onlyKnownCaps := configv1.ClusterVersionCapabilitiesStatus{knownCapabilities, knownCapabilities}
```

Comment:
Can we use explicit properties here:

```golang
onlyKnownCaps := configv1.ClusterVersionCapabilitiesStatus{
  EnabledCapabilities: knownCapabilities, // enabled-caps filtering happens at apply time, so don't filter on that here
  KnownCapabilities:   knownCapabilities,
}
```

Or some such?  I don't expect `ClusterVersionCapabilitiesStatus` to be all that volatile, but if it does add additional properties in the future, the explicit references here seem like they'd make life easier on the future dev adjusting this.